File: joey/week3-220728/2114. Maximum Number of Words Found in Sentences.py
class Solution1:
    def mostWordsFound(self, sentences: List[str]) -> int:
        maximum = 0
        for s in sentences:
            count = s.count(' ') + 1
            if (count > maximum):
                maximum = count
        return maximum

class Solution2:
    def mostWordsFound(self, sentences: List[str]) -> int:
        return max([(s.count(' ') + 1) for s in sentences])

File: billy/week3-220728/2114. Maximum Number of Words Found in Sentences.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function(sentences) {
    let max = [];
    let count = 0;

    for (let i = 0; i < sentences.length; i += 1) {
        for (let j = 0; j < sentences[i].length; j += 1) {
            if (sentences[i][j] === ' ') count++;
        }

        max.push(count);
        count = 0;
    }

    return Math.max(...max) + 1;
};